src/theme/Footer/index.tsx

```diff
@@ -108,6 +108,11 @@ function Footer(): JSX.Element | null {
                 <li>
                   <Link to="/">Documentation</Link>
                 </li>
+                <li>
+                  <Link href="https://minaprotocol.com/join-zkignite-cohort-2">
+                    zkIgnite, Cohort 2
+                  </Link>
+                </li>
```

Comment:
```suggestion
```

I like the way that you are thinking @LucidSamuel  _but_ the docs site is not the right place to include time-bound links like this cohort 2, so let's remove it. The MF website keeps their page footer up to date.

@drea-rodriguez  it would be so handy to have an evergreen page that talks about zkIgnite in general 

We follow https://developers.google.com/style/timeless-documentation

Reply:
Okay great, an evergreen link that supports or shows the previous, current, and upcoming cohort would be great.